skrub/_dataframe/_common.py

```diff
@@ -800,6 +818,21 @@ def _drop_nulls_polars(column):
     return column.drop_nulls()
 
 
+@dispatch
+def fill_nan(obj, value):
+    raise NotImplementedError()
+
+
+@fill_nan.specialize("pandas")
+def _fill_nan_pandas(obj, value):
+    return obj.fillna(value)
+
+
+@fill_nan.specialize("polars")
+def _fill_nan_polars(obj, value):
+    return obj.fill_nan(value)
```

Comment:
do we want fill nan or fill nulls?

Reply:
and I don't see this function being used anywhere?

Reply:
We want pandas' `fillna` -> polars' `fill_nan` + `fill_null` since polars distinguishes NA/NAN but pandas doesn't

Let's rename it `fill_null` to be more consistent with the `drop_nulls`, `has_nulls`, etc